one/src/lib.rs

```diff
@@ -149,6 +149,14 @@ struct DaemonOpts {
         env = "CERAMIC_ONE_IDLE_CONNS_TIMEOUT_MS"
     )]
     idle_conns_timeout_ms: u64,
+
+    /// Enable authentication feature
+    #[arg(
+        long,
+        default_value_t = false,
+        env = "CERAMIC_ONE_ENABLE_AUTHENTICATION"
+    )]
+    enable_authentication: bool,
```

Comment:
didn't think about it first pass, but wonder if we'd want to make features flags and enum we could pass multiple times or as a comma separated string... A bool seems find for now, though we should probably mark these things unstable before GA.